File: f#/2025/day00.fs
namespace _2025

open Xunit

module Day00 =
    let Part1 () =
        let lines = Utils.ReadInputLines("day00.txt")
        lines |> Array.map int |> Array.sum


    [<Fact>]
    let Test1 () = Assert.Equal(110, Part1())

File: c#/2025/day00.cs
namespace _2025;

public static class Day00
{
    [Fact]
    public static void Test1()
    {
        Assert.Equal(110, Part1());
    }

    public static int Part1()
    {
        var lines = Utils.ReadInputLines("day00.txt");
        return lines.Select(int.Parse).ToList().Sum();
    }
}
